src/qibolab/instruments/dummy.py

```diff
@@ -9,6 +9,7 @@
 from qibolab.instruments.abstract import AbstractInstrument
 from qibolab.platform import Qubit
 from qibolab.pulses import PulseSequence, PulseType
+from qibolab.result import IntegratedResults, SampleResults
 from qibolab.sweeper import Parameter, Sweeper
```

Comment:
I don't think we use them

```suggestion
from qibolab.pulses import PulseSequence
from qibolab.sweeper import Sweeper
```